src/smart-components/role/add-role-new/add-role-wizard.js

```diff
@@ -61,6 +63,8 @@ const AddRoleWizard = ({ pagination }) => {
     container.current.hidden = cancelWarningVisible;
   }, [cancelWarningVisible]);
 
+  const onClose = () => history.push(`${paths.roles}${createQueryParams({ page: 1, per_page: pagination.limit })}`);
```

Comment:
If it won't refactoring other parts of the code already present in this PR, can we use the object argument instead of constructing the whole URL string?

```jsx
history.push({pathname, search, hash})
```